src/Okta.Idx.Sdk/IdxClient.cs

```diff
@@ -281,6 +281,16 @@ public async Task<TokenResponse> RedeemInteractionCodeAsync(IIdxContext idxConte
         {
             try
             {
+                if (string.IsNullOrEmpty(interactionCode))
+                {
+                    throw new ArgumentNullException("interactionCode", "Interaction code was not specified.");
+                }
+
+                if (string.IsNullOrEmpty(idxContext.CodeVerifier))
+                {
+                    throw new ArgumentNullException("idxContext.CodeVerifier", "CodeVerifier was not specified.");
```

Comment:
Nit: Can we use `nameof(idxContext.CodeVerifier)` instead? . Same with `interactionCode` above.